parsec/interfaces/ptg/ptg-compiler/jdf2c.c

```diff
@@ -6983,7 +6999,7 @@ static void jdf_generate_code_hook(const jdf_t *jdf,
     coutput("#if defined(PARSEC_HAVE_DEV_%s_SUPPORT)\n", type_upper);
     if( NULL != type_property) {
 
-        if (!strcasecmp(type_property->expr->jdf_var, "cuda")
+        if (!strncasecmp(type_property->expr->jdf_var, "cuda", 4)  /* for batched */
```

Comment:
Looks like a leftover from a prior iteration of that patchset that used the `type=cuda_batched` instead of adding a new `batched` property.

I assume the expectation is that we can have `batched` and `non batched` CUDA bodies simultaneously. Did you test this works?